test/extensions/secondorder/secondorder_settings.py

```diff
@@ -70,6 +70,63 @@
 ###############################################################################
 
 SECONDORDER_SETTNGS += [
+    {
+        "input_fn": lambda: torch.rand(3, 3, 7),
+        "module_fn": lambda: torch.nn.Sequential(
+            torch.nn.Conv1d(3, 2, 2),
+            torch.nn.ReLU(),
+            torch.nn.Flatten(),
+            torch.nn.Linear(12, 5),
+        ),
+        "loss_function_fn": lambda: torch.nn.CrossEntropyLoss(reduction="sum"),
+        "target_fn": lambda: classification_targets((3,), 5),
+    },
+    {
+        "input_fn": lambda: torch.rand(3, 3, 7),
+        "module_fn": lambda: torch.nn.Sequential(
+            torch.nn.Conv1d(3, 2, 2, bias=False),
+            torch.nn.ReLU(),
+            torch.nn.Flatten(),
+            torch.nn.Linear(12, 5),
+        ),
+        "loss_function_fn": lambda: torch.nn.CrossEntropyLoss(reduction="sum"),
+        "target_fn": lambda: classification_targets((3,), 5),
+    },
```

Comment:
Meta-comment: One pro of the new test suite is that it is easy to add new tests explicitely. Now we are at a stage where it is hard to me to figure out the test coverage by staring at the big dictionary. I am not saying that we should get rid of all explicit test settings, but maybe establish a subset of problems (for instance a small network `Flatten -> Linear -> Activation -> Linear -> Loss` and a small CNN) which are then built with and without bias and with different activation functions

Reply:
Among the line of thoughts above: Maybe something like
```python

def make_simple_cnn_fn(act_cls):
    def make_simple_cnn():
        return torch.nn.Sequential(
            torch.nn.Conv2d(...),
            act_cls(),
            torch.nn.Flatten(),
            torch.nn.Linear(...),
        ),
    
    return make_simple_cnn
```

and loop over different activation functions to create the entries in `SECOND_ORDER_SETTINGS`.

Reply:
Please use this approach to add such a simple CNN test for the new activations `ELU`, ...

Reply:
Where should this function go? Somewhere central so we can call this function right?
How about `test/networks.py`?

Reply:
`test/extensions/automated_settings.py`